src/helpers/i18n/en/translation.json

```diff
@@ -164,7 +161,7 @@
     "SEE_ALL_RESULTS": "See all results for '{{searchQuery}}'"
   },
   "LOADING": {
-    "LOGGIN_IN": "Logging in...",
+    "LOGGING_IN": "Logging in...",
```

Comment:
Please also change in the referring file:
components/login/login.component.js

And in the Spanish translation:
src/helpers/i18n/es/translation.json